gitlab-pipeline/stage/publish.yml

```diff
@@ -147,9 +147,18 @@ publish:mender-dist-packages:
   rules:
     - if: '$INTEGRATION_REV =~ /^[0-9]+\.[0-9]+\.[0-9]+(-build\d+)?$/'
   variables:
+    # Unset DOCKER_ variables, the pipeline will execute on shared runners
+    DOCKER_HOST: ""
+    DOCKER_CERT_PATH: ""
+    DOCKER_TLS_VERIFY: ""
+    DOCKER_TLS_CERTDIR: ""
```

Comment:
To be honest, I think it's a bad idea to keep these variables within the `.gitlab-ci.yml` (even though it was me who introduced them in the first place).
It's better to define them statically in the [gitlab-runner configuration](https://gitlab-docs.creationline.com/runner/configuration/advanced-configuration.html#the-runnersdocker-section) for the relevant build hosts so we avoid polluting the runner's environment.

Reply:
We need to define them per job (i.e. integration tests cannot have them set), so that won't work if we set them in the gitlab-runner configuration